action.yml

```diff
@@ -26,6 +26,10 @@ inputs:
     description: "Run the action in debug mode and get debug logs printed in console"
     required: false
     default: "false"
+  aggregate-mode:
```

Comment:
Do we really need this?
I yet to try configuring "jacoco-playground" repo to generate aggregated report. But from your test cases, I saw the  XML of the generated file. Can't we just check for the presence of "<group>" and then figure out it's in aggregate-mode?

Reply:
In the DTD for the jacoco report xml, we have "<group>" which the github-action currently doesn't support. I will use your PR to add support for that. I am going to remove the "aggregate-mode" flag, and keep the rest.